assets/sass/config/_variables.scss

```diff
@@ -311,3 +311,6 @@ $mdc-text-field-label: $c-interactive-on-disable-container;
 
 // Custom max width for settings notice content
 $max-width-settings-notice-content: 1098px;
+
+// Custom z-index values
```

Comment:
As @aaemnnosttv alluded to [here](https://github.com/google/site-kit-wp/issues/6907#issuecomment-1634979154), it's probably worth adding a bit more info as a comment to describe the notable, related z-indexes that you found while investigating this issue. No need to go into too much detail - we can always refer back to this issue for the full writeup - but a brief summary would be handy, essentially these bits:

> * The top admin bar and its child menus, at z-index `99999`
> * The left most main/side admin menu wrapper, at z-index `9990`
> * The left most main/side admin menu sub menus on hover, at z-index `9999`